client/elife-theme/src/elements/Menu.js

```diff
@@ -0,0 +1,45 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+export default {
+  Arrow: css`
+    color: ${th('colorTextSecondary')};
+    transition: none;
+  `,
+  ArrowContainer: css`
+    border-left: none;
+  `,
+  Opener: css`
+    &:hover {
+      border-color: ${th('colorBorder')};
+    }
+    ${props =>
+      props.open &&
+      `border-bottom-left-radius: 0; border-bottom-right-radius: 0;`};
+  `,
+  Option: css`
+    height: ${th('space.5')};
+    line-height: ${th('space.5')};
```

Comment:
Has there been a decision to use `space` instead of the `fontLineHeightXYZ` variables in our theme?

Reply:
In this instance line-height is being used for layout (to centre the text in its box) rather than typography.